contribute/style-guides/redux.md

```diff
@@ -117,17 +117,48 @@ Type '{ nonExistingProperty: string; data: string[]; }' is not assignable to typ
   Object literal may only specify known properties, and 'nonExistingProperty' does not exist in type 'ExampleReducerState'. [2322]
 ```
 
-## New test functionality
+## Test functionality
 
 ### reducerTester
 
 Fluent API that simplifies the testing of reducers
 
-#### Example
+#### Simple Example
 
 ```typescript
 reducerTester()
   .givenReducer(someReducer, initialState)
   .whenActionIsDispatched(someAction('reducer tests'))
   .thenStateShouldEqual({ ...initialState, data: 'reducer tests' });
 ```
+
+#### Complex Example
+Sometimes you encounter a `resulting state` that contains properties that are hard to compare like `Dates` but you still 
+want to compare that other props in state are correct.
+
+Then you can use `thenStatePredicateShouldEqual` function on `reducerTester` that will return the `resulting state` that
+you can expect upon.
```

Comment:
This line seems to be hard wrapped. It should be on the same line as the one above.

Reply:
Will fix that!